spec/color_ls/flags_spec.rb

```diff
@@ -325,4 +325,156 @@
+
+    before do
+      fileInfo = instance_double(
+        'FileInfo',
+        :group => "sys",
+        :mtime => Time.now,
+        :directory? => false,
+        :owner => "user",
+        :name => "a.txt",
+        :show => "a.txt",
+        :nlink => 1,
+        :size => 128,
+        :blockdev? => false,
+        :chardev? => false,
+        :socket? => false,
+        :symlink? => false,
+        :stats => OpenStruct.new(
+          mode: 0o444, # read for user, owner, other
+          setuid?: true,
+          setgid?: true,
+          sticky?: true
+        ),
+        :executable? => false
+      )
+
+      allow(ColorLS::FileInfo).to receive(:new).with("#{FIXTURES}/a.txt", link_info: true) { fileInfo }
+    end
+
+    it 'lists with group info' do
+      expect { subject }.to output(/sys/).to_stdout
+    end
+    it 'lists without user info' do
+      expect { subject }.not_to output(/user/).to_stdout
+    end
+  end
+
+  context 'with -o and -g flag' do
+    let(:args) { ['-og', "#{FIXTURES}/a.txt"] }
+
+    before do
+      fileInfo = instance_double(
+        'FileInfo',
+        :group => "sys",
+        :mtime => Time.now,
+        :directory? => false,
+        :owner => "user",
+        :name => "a.txt",
+        :show => "a.txt",
+        :nlink => 1,
+        :size => 128,
+        :blockdev? => false,
+        :chardev? => false,
+        :socket? => false,
+        :symlink? => false,
+        :stats => OpenStruct.new(
+          mode: 0o444, # read for user, owner, other
+          setuid?: true,
+          setgid?: true,
+          sticky?: true
+        ),
+        :executable? => false
+      )
+
+      allow(ColorLS::FileInfo).to receive(:new).with("#{FIXTURES}/a.txt", link_info: true) { fileInfo }
+    end
+
+    it 'lists without group info' do
+      expect { subject }.not_to output(/sys/).to_stdout
+    end
+    it 'lists without user info' do
+      expect { subject }.not_to output(/user/).to_stdout
+    end
+  end
+
+  context 'with -G flag in a listing format' do
+    let(:args) { ['-l', '-G', "#{FIXTURES}/a.txt"] }
+
+    before do
+      fileInfo = instance_double(
+        'FileInfo',
+        :group => "sys",
+        :mtime => Time.now,
+        :directory? => false,
+        :owner => "user",
+        :name => "a.txt",
+        :show => "a.txt",
+        :nlink => 1,
+        :size => 128,
+        :blockdev? => false,
+        :chardev? => false,
+        :socket? => false,
+        :symlink? => false,
+        :stats => OpenStruct.new(
+          mode: 0o444, # read for user, owner, other
+          setuid?: true,
+          setgid?: true,
+          sticky?: true
+        ),
+        :executable? => false
+      )
+
+      allow(ColorLS::FileInfo).to receive(:new).with("#{FIXTURES}/a.txt", link_info: true) { fileInfo }
+    end
+
+    it 'lists without group info' do
+      expect { subject }.not_to output(/sys/).to_stdout
+    end
+    it 'lists without user info' do
```

Comment:
The check is correct, but the description says the opposite.
```suggestion
    it 'lists with user info' do
```